resources/js/Pages/Layout.vue

```diff
@@ -40,24 +40,24 @@
             <section>
                 <h2 class="h5">{{ $i18n('mismatch-finder-footer-more-tools') }}</h2>
                 <p>
-                    <wikit-link href="https://query.wikidata.org/querybuilder/">
+                    <a href="https://query.wikidata.org/querybuilder/">
```

Comment:
The margins are collapsed in the footer too, but I see them collapsed only in the results page. o_O

Development: 
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/1c6144ca-416f-4a62-9a72-3411777950b3)

This branch:
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/76aa525f-3e74-413e-83b1-38ebe41dbd3d)

Reply:
Yeah the styles were on h2 by mistake. it should look better now